Helpers/ServiceResponse.cs

```diff
@@ -1,6 +1,17 @@
 ﻿namespace ToggleBuddy.API.Helpers
 {
-    public class ApiResponse<T>
+    public enum ResponseStatus
+    {
+        Success,
+        Error,
+        Processing,
+        NotFound,
+        Unauthorized,
```

Comment:
Personally I don't really get the idea of this "ServiceResponse"

Reply:
We don't handle the http request in the service layer but the controller. It's reusable for the service layers, that is why we made it as an utility.